zanata-client-commands/src/main/java/org/zanata/client/commands/init/SourceConfigPrompt.java

```diff
@@ -191,4 +218,92 @@ public String getExcludes() {
     public Set<String> getDocNames() {
         return docNames;
     }
+
+    private SrcDocsFinder makeSrcDocFinder() {
+        String projectType = pushOptions.getProjectType();
+        if (projectType.equalsIgnoreCase(File.name())) {
+            return new RawSrcDocsFinder(pushOptions);
+        } else {
+            return new OtherSrcDocsFinder(pushOptions);
+        }
+    }
+
+    interface SrcDocsFinder {
```

Comment:
Is this repeating the logic in the actual push/pull commands?
This and its implementations are big enough that I would prefer them in their own file. If possible it would be even better if these can be re-used both here and in the other commands. If the code is in 2 different places, there is a risk that the files found in the init command will not match the files found in push/pull.